docs/user-guide/street-view.md

```diff
@@ -0,0 +1,23 @@
+# Street View
+*******************
+
+The **Street View** is an extension that can be included in the third step of the application context wizard to allow the end user to browse data from Google street view. The <img src="../img/button/street-view-button.jpg" class="ms-docbutton"/> button, present in *Burger Menu* <img src="../img/button/burger.jpg" class="ms-docbutton" />, provides to the user the pop-up to navigate the map with Google street view. 
+
+<img src="../img/street-view/streetview-popup.jpg" class="ms-docimage" width="500px"/>
+
+Once the *Street View* pop-up opens, the streets on the map are highlighted and the user can select one on the map.
```

Comment:
When the tool is activated, a window opens and the streets highlighted on the map so that the user can select one of them with a simple click of the mouse.